### ISSUE TITLE
Demande de dépublication d'une fiche de MD

### ISSUE THREAD

**Issue by rr-pro** [etalab/geo.data.gouv.fr#388]
Bonjour,
la fiche de métadonnées "Monuments Historiques surfacique générateur de servitude AC1 dans le Gard" est présente sur la passerelle Inspire en https://inspire.data.gouv.fr/datasets/333f77ba41
après son moissonnage depuis Géo-IDE.

Depuis, son auteur l'a dépubliée de Géo-IDE. La propagation de la suppression s'est faite vers le Géocatalogue mais pas vers la passerelle Inspire.

Son auteur demande qu'elle soit supprimée sur la passerelle Inspire. Pouvez vous vous en charger ?

Merci par avance,

Robert Rivière - MEEM

**Comment by jdesboeufs** [comment]
Bonjour,

Y-a-t'il un motif à cette suppression ? Une fiche de remplacement ?

**Comment by rr-pro** [comment]
Bonjour Jérôme,
s'agissant des SUP, je présume que les DDT sont désormais très rigoureuses sur la publication d'infos vis à vis du Géoportail de l'Urbanisme.
Cette fiche a du être par erreur temporairement visible sur internet puis sa visibilité a été limitée à l'intranet MEEM.
Cordialement

**Comment by jdesboeufs** [comment]
<img width="850" alt="capture d ecran 2017-05-23 a 16 07 42" src="https://cloud.githubusercontent.com/assets/1231232/26359339/7debaf7a-3fd5-11e7-8bb1-48bf935fc97b.png">

Voici le genre de conséquence que peut avoir la dépublication de données en amont.
Ça renforce notre volonté d'archiver les données quand elles transitent par notre plateforme.

Je vais voir pour planifier une resynchro massive ASAP.

**Comment by rr-pro** [comment]
Il y a apparemment un certain flou autour du caractère opérationnel ou pas du Géoportail de l'urbanisme et du qui fait quoi, ce qui ne simplifie pas la mise à disposition de la donnée... A mon avis les catalogues généralistes (et leurs utilisateurs) n'en sont pas la cause mais en subissent les conséquences.

**Comment by jdesboeufs** [comment]
Oui, et d'ailleurs le GPU est tout à fait capable de moissonner les catalogues, donc les services devraient continuer à déposer les données comme avant.

**Comment by rr-pro** [comment]
Oui, mais pas exactement comme avant cependant :

- pour certaines catégories de données, les services du MEEM n'en sont 
plus responsables et ne doivent plus les mettre en ligne à la place d'un 
autre acteur (typiquement commune ou EPCI), directement ou indirectement 
par moissonnage

- le statut de la donnée est devenu important, il ne faut probablement 
plus diffuser les données "en cours d'élaboration", ce qui doit être le 
cas précisément du jeu à l'origine de cette discussion

**Comment by jdesboeufs** [comment]
Oui. Mais ce qui est dommage c'est que dans le cas des Monuments historiques, que les DDT dépublient massivement sans que les DRAC soient prêtes à prendre le relais, ça créé un énorme vide qui peut durer des années.

**Comment by rr-pro** [comment]
A ce moment là, si tu constates un phénomène massif, je peux de mon côté 
faire un retour vers l'équipe MEEM qui pilote les relations avec le GPU 
pour les en informer.
T'es-t'il possible de retrouver des cas concrets de ce type de problème ?

**Comment by jjacq44** [comment]
Bonjour, J'ai un cas de figure un peu similaire de dépublication souhaitée. Nous avions commencé à préparer deux fiches de métadonnées relatives à la SUP PM3 qui ont été publiées un peu vite (sans que la donnée soit disponible et sans qu'une décision ne soit prise sur le service  producteur légitime (DREAL ou DDTM),  Nous avons dépublié de Géo-IDE les 2 fiches mais elle restent présentes sur la passerelle. Cela n'apporte rien, me semble-t-il, pour un utilisateur de laisser ces fiches puisque la donnée n'a jamais été accessible. Le mieux serait de les supprimer.
Hors problématique GPU, j'ai également une fiche de métadonnée présente sur la passerelle intitulée "Service pénitentiaire d'insertion et de probation - Loire-Atlantique". La DDTM n'ayant aucune légitimité sur cette donnée et la donnée n'étant plus à jour, elle a été dépubliée mais elle reste accessible sur la passerelle. Idem pas d'apport pour un utilisateur potentiel.

**Comment by rr-pro** [comment]
Bonjour,

un 'up' sur ce sujet, nous avons également été à nouveau sollicités par la DREAL BFC au sujet du jeu https://geo.data.gouv.fr/fr/datasets/09a62430493b20e0f3fdbc1ae23b91c263a4e2b5

La DREAL nous explique que suite à la fusion des régions, les données déposées par la DREAL le sont désormais dans la plateforme régionale (IdéoBFC), et celles présentes sur Géo-IDE sont dépubliées.

Le jeu de données cité est bien taggé "Idéo BFC", mais aussi toujours Géo-IDE, et les URLs de téléchargement ou WxS présentées référencent celles de Géo-IDE qui n'existent plus.

Merci
